specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-09-01-preview/definitions.json

```diff
@@ -3605,6 +3609,10 @@
           "type": "string",
           "description": "Client secret of the Application in external Identity Provider, used to authenticate login request. For example, it is App Secret for Facebook login, API Key for Google login, Public Key for Microsoft. This property will not be filled on 'GET' operations! Use '/listSecrets' POST request to get the value.",
           "minLength": 1
+        },
+        "certificateId": {
+          "type": "string",
+          "description": "Certificate ID of the Application in the external Identity Provider, used to authenticate login request."
```

Comment:
type: x-ms-arm-id

/subscriptions/sid/resourcegroups/rg/providers/Microsoft.ApiManagement/service/,,,,

Reply:
Use the file in 2024-06-01-preview